pkg/local_object_storage/writecache/writecache.go

```diff
@@ -59,6 +59,11 @@ type cache struct {
 	mode    mode.Mode
 	modeMtx sync.RWMutex
 
+	// error handling while flushing.
+	flushErrCh chan error
+	retryMtx   sync.RWMutex
+	lastError  error
```

Comment:
```suggestion
	lastFlushError  error
```
(if we'll decide to keep it)